backend/trojan_dining/views.py

```diff
@@ -67,3 +67,21 @@ def get(self, request):
         del retrieved_menu_dict['_state']
         return JsonResponse({"Menu": retrieved_menu.__dict__})
 
+class PostSubscription(APIView):
```

Comment:
Nit: lets rename this to `class CreateNotification` - its not super useful to include the request method in the class name 😅